python/pyarrow/_rados.pyx

```diff
@@ -22,6 +22,17 @@ from pyarrow.lib cimport *
 from pyarrow.lib import frombytes, tobytes
 
 cdef class RadosParquetFileFormat(FileFormat):
+    """
+    A ParquetFileFormat implementation that offloads the fragment
+    scan operations to the Ceph OSDs
+
+    Parameters
+    ---------
+    ceph_config_path: The path to the ceph config file.
+    data_pool: Name of the CephFS data pool.  
+    user_name: Username to access the Ceph data cluster.
```

Comment:
Another way to say it will be "The username accessing the Ceph cluster".

Reply:
Done.